update pre commit config for terraform_fmt & added app_gateway_backend_check (#1197)
hmcts/azure-platform-terraform: README.md

# common-platform-terraform
All things terraform

## Pre-commit Installation

> This repository has been configured with pre-commit hooks. To install pre-commit hook, see [here](https://github.com/hmcts/pre-commit-hooks#pre-commit-installation)

Reviewer: ```suggestion
This repository has been configured with pre-commit hooks. To install pre-commit see its [installation guide](https://pre-commit.com/#installation)

```

> Additional for this repository, additional dependencies: 

Reviewer: ```suggestion
For this repository you will also need these dependencies: 

```

> - terraform [Installation here](https://learn.hashicorp.com/tutorials/terraform/install-cli)
> - yq [Installation here](https://github.com/mikefarah/yq#install)

Reviewer: ```suggestion
- [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
- [yq](https://github.com/mikefarah/yq#install)
```
